app/assets/scripts/views/about.js

```diff
@@ -138,47 +141,31 @@ var About = React.createClass({
                   <p>Mauris posuere mauris a molestie ultrices. Donec risus ligula, rutrum laoreet elementum aliquam, ullamcorper in nibh.</p>
                 </div>
               </header>
-              <ul className='board-list'>
-                <li>
-                  <article className='board-member'>
-                    <figure className='board-member__avatar'>
-                      <img src='/assets/graphics/content/board/board-avatar--christa-hasenkopt.jpg' width='320' height='320' alt='Board avatar' />
-                    </figure>
-                    <h1 className='board-member__title'>Christa Hasenkopf</h1>
-                    <p className='board-member__role'>Chief Executive Officer</p>
-                  </article>
-                </li>
 
-                <li>
-                  <article className='board-member'>
-                    <figure className='board-member__avatar'>
-                      <img src='/assets/graphics/content/board/board-avatar--placeholder.jpg' width='320' height='320' alt='Board avatar' />
-                    </figure>
-                    <h1 className='board-member__title'>Matthew Higgins</h1>
-                    <p className='board-member__role'>Treasurer</p>
-                  </article>
-                </li>
+              <TeamList items={_(content.governingBoard)
```

Comment:
Since this is static content, you should sort outside the render function as to avoid creating new objects every time.